skfuzzy/control/tests/test_controlsystem.py

```diff
@@ -53,7 +53,7 @@ def test_tipping_problem():
     tip_sim2.compute()
 
     assert tip_sim.output == tip_sim2.output
-    tst.assert_allclose(tip_sim.output['tip'], 20.244508118433625)
+    tst.assert_allclose(tip_sim.output['tip'], 19.847607362)
```

Comment:
This change is not negligible (it's nearly 7% lower relative to the prior), and I hand-calculated this result using the old manual method before the new API was created. Does this agree with the output from Fuzzy Jess and/or other packages?